hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/om/helpers/OmBucketInfo.java

```diff
@@ -366,7 +367,6 @@ public static OmBucketInfo getFromProtobuf(BucketInfo bucketInfo) {
     return obib.build();
   }
 
-  @Override
```

Comment:
Why do we remove the @Override for equals?

Reply:
Sorry mistakenly removed it. Reverted it back.